app/src/main/java/com/simplemobiletools/draw/MyCanvas.java

```diff
@@ -95,17 +105,17 @@ protected void onDraw(Canvas canvas) {
         changePaint(mPaintOptions);
         canvas.drawPath(mPath, mPaint);
 
-        if(!mIsSaving) {
+        if(mIsStrokeWidthBarEnabled && !mIsSaving) {
             drawPreviewDot(canvas);
         }
     }
 
     private void drawPreviewDot(Canvas canvas) {
         mPaint.setColor(Utils.shouldUseWhite(mPaintOptions.color)?Color.WHITE:Color.BLACK);
         mPaint.setStrokeWidth(100);
-        canvas.drawPoint(getWidth()/2, getHeight() - 100, mPaint);
+        canvas.drawPoint(getWidth()/2, getHeight() - 120, mPaint);
```

Comment:
also try changing the background circle width dynamically too, so its always just a thin line around the picked color. Its a too big circle if the selected paint is small